test/CMakeLists.txt

```diff
@@ -65,13 +65,13 @@ set_tests_properties(oqs_tlssig
     # Some algorithms known to overflow OpenSSL limits; TBD/resolved upstream
     PROPERTIES ENVIRONMENT "OPENSSL_MODULES=${CMAKE_BINARY_DIR}/lib;OQS_SKIP_TESTS=picnicl1full,rainbowVclassic,sphincs"
 )
-add_executable(oqs_test_tlssig oqs_test_tlssig.c test_common.c ${OPENSSL_BLDTOP}/test/helpers/ssltestlib.c)
+add_executable(oqs_test_tlssig oqs_test_tlssig.c test_common.c tlstest_helpers.c)
 # e_os.h resides in top-level for OSSL3:
```

Comment:
Are all of these include directories still needed? `e_os.h` should not be required any more, right?